scripts/wsclient-test-generator/TestGeneratorAgent.py

```diff
@@ -154,8 +204,9 @@ def parse_arguments():
     
     # File system navigation and search
     "shell(cd)",            # CRITICAL - Change directories
-    "shell(bash)",          # CRITICAL - General shell access
-    "shell(sh)",            # CRITICAL - General shell access
+    "shell",                # CRITICAL - General shell access (enables &&, |, redirection)
```

Comment:
in general, I'm wondering if we should prompt the agent to notify us when it would like access to anything not white listed, and have it provide justification and alternatives.